compiler/one-cmds/one-build.cfg

```diff
@@ -0,0 +1,79 @@
+# This configuration file is used to configure the parameters and initial settings for ONE command driver.
+
+# NOTE You must specify the parameters for the driver you want to run, 
+#      and other options are left blank.
+#
+# NOTE Sometimes you need the intermediate output of each driver. 
+#      You can get it by specifying each driver's output path(*_output_path), i.e., import_tf_output_path.
+
+# input_path                    Path to input
+input_path="/path/to/incepV3/inception_v3.pb"
+# output_path                   Path to output
+output_path="inception_v3.circle"
+
+# driver to use
+use_one_import_tf="true"
+use_one_import_bcq="false"
+use_one_import_tflite="false"
+use_one_optimize="true"
+use_one_quantize="false"
+use_one_pack="true"
+use_one_codegen="false"
```

Comment:
I reordered `use` options to make it more readable. Users first select what driver to use and then fill the rest of other options. If users miss some options, the script occurs an error.